src/Dropdown.js

```diff
@@ -263,15 +265,7 @@ class Dropdown extends Component {
         >
           {value}
           <div className={`${baseClassName}-arrow-wrapper`}>
-            {arrowOpen && arrowClosed ? (
-              this.state.isOpen ? (
-                arrowOpen
-              ) : (
-                arrowClosed
-              )
-            ) : (
-              <span className={arrowClass} />
-            )}
+            {arrowOpen && arrowClosed ? arrow : <span className={arrowClass} />}
```

Comment:
A componentized arrow and/or a unit test would be good as well. Even without those, this is a good change to merge in.

Reply:
can make it a component in another PR.
this change is related to eslint rule changes

Reply:
ok let's merge :)